src/main/java/duke/exception/InvalidDeadlineException.java

```diff
@@ -0,0 +1,17 @@
+package duke.exception;
+
+/**
+ * Represents an InvalidDeadlineException and handles the exceptions related
+ * to invalid deadline tasks.
+ */
+public class InvalidDeadlineException extends InvalidInputException {
```

Comment:
I like how you clearly named each of the exceptions from which I can instantly get the meaning!

Reply:
Thanks!